dts/riscv/it8xxx2.dtsi

```diff
@@ -808,6 +809,30 @@
 			pinctrl-0 = <&pinctrl_pwm7>; /* GPA7 */
 			#pwm-cells = <2>;
 		};
+		tach0: tach@f0181e {
+			compatible = "ite,it8xxx2-tach";
+			reg = <0x00f0181e 1   /* F1TLRR */
+			       0x00f0181f 1   /* F1TMRR */
+			       0x00f01848 1>; /* TSWCTLR */
+			dvs-bit = <0x8>;   /* Bit(3) */
+			chsel-bit = <0x4>; /* Bit(2) */
+			label = "TACH_0";
+			status = "disabled";
+			pinctrl-0 = <&pinctrl_tach0a   /* GPD6 */
+				     &pinctrl_tach0b>; /* GPJ2 */
```

Comment:
this belongs to board-level DT, will likely need to be addressed when porting to pinctrl.

Reply:
Ack